lib/mix/lib/mix/tasks/release.ex

```diff
@@ -0,0 +1,916 @@
+defmodule Mix.Tasks.Release do
+  @moduledoc """
+  Assembles a release for the current project:
+
+      MIX_ENV=prod mix release
+      MIX_ENV=prod mix release NAME
+
+  Once a release is assembled, it can be packaged and deployed to a
+  remote host, as long as the remote host runs on the same operating
+  system (OS) and versions as the machine running the `mix release`
```

Comment:
"versions" is a bit ambiguous to me here. Maybe it should clarify "Elixir and Erlang/OTP versions"?

Reply:
Definitely ambiguous as it is the exact same OS version. :D How to better phrase this? "The same operating system (OS) distribution and version"?

Reply:
I think that is a bit more clear. Should we also mention that the `ERTS` version must match if you have chosen not to include it in the release?

Reply:
We do mention that but in a separate section (when we talk about include_erts: false).

Reply:
Done!